dockers/docker-sonic-bmp/critical_processes

```diff
@@ -0,0 +1,4 @@
+group:sonic-bmp
+program:openbmpd
+program:bmpcfgd
+{%- endif %}
```

Comment:
>endif [](http://example.com/codeflow?start=4&length=5)

did you miss `if` statement? did you test?

Reply:
fixed